letsencrypt/plugins/manual.py

```diff
@@ -57,7 +57,8 @@ class ManualAuthenticator(common.Plugin):
 cd {root}/public_html
 echo -n {validation} > {response.URI_ROOT_PATH}/{encoded_token}
 # run only once per server:
-python -c "import BaseHTTPServer, SimpleHTTPServer; \\
+$(command -v python2 || command -v python2.7 || command -v python2.6) -c \\
```

Comment:
Have you considered using `type -P` instead (looks also for keywords, aliases, functions)?